app/models/event.rb

```diff
@@ -20,6 +20,62 @@ class Event < ApplicationRecord
     in_week(week).group_by { |e| e.start_time.wday }
   }
 
+  class << self
+    def not_free(week)
+      css_color = 'repeating-linear-gradient(-45deg,#D3E9FF,#D3E9FF 10px,#B8DCFF 10px,#B8DCFF 20px)'
+      occupied_slot = ConferenceRoom.new(capacity: 99, title: 'NoWayLand', color: css_color, email: 'fake_email@com')
```

Comment:
Do we need to pass such params `email: fake_email@com`? Does this object need to be valid?